examples/jsm/nodes/core/AttributeNode.js

```diff
@@ -91,9 +92,15 @@ class AttributeNode extends Node {
 
 		} else {
 
-			console.warn( `AttributeNode: Attribute "${ attributeName }" not found.` );
+			console.warn( `AttributeNode: Vertex attribute "${ attributeName }" not found on geometry.` );
 
-			return builder.generateConst( nodeType );
+			let value = undefined;
+
+			// Color attribute fallback should be white, not black.
+			if ( attributeName === 'color' )
```

Comment:
I think we can move this fallback to `VertexColorNode`, overriding `generate()` function, something like:

```js
generate( builder ) {

	const geometryAttribute = builder.hasGeometryAttribute( attributeName );
	
	let result;

	if ( geometryAttribute === true ) {

		result = super.generate( builder );

	} else {
	
		// fallback

		result = builder.generateConst( this.nodeType, new Vector4( 1, 1, 1, 1 ) );

	}
	
	return result;

}
```